.travis.yml

```diff
@@ -37,8 +37,9 @@ install:
   - sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-6 90
   - sudo service mysql stop
 
+# TODO(lwsanty): we probably need to include go mod case into our CI on 'make dependencies' command to avoid this fixes in the future
 script:
-  - make dependencies
+  - go mod download
```

Comment:
instead of that, just adding `GO111MODULE=on` environment variable makes everything else work seamlessly.

Reply:
Done